system-int-test/test/cmr/system_int_test/search/collection_spatial_keyword_search_test.clj

```diff
@@ -31,7 +35,18 @@
            "LA" [coll4]
            ["Detroit"] [coll5 coll7]
            ["LL" "Detroit"] [coll5 coll6 coll7]
-           "BLAH" []))
+           "Gaza Strip" [coll-gaza]
+           "Tuolumne River Basin" [coll-iso-sample]
+           "North America" [coll-iso-sample]))
+    (testing "search by spatial keywords using wildcard * for cmr-4192 collection"
+      (is (d/refs-match? [coll-iso-sample]
+                         (search/find-refs :collection
+                                           {:spatial-keyword "*river*"
+                                            "options[spatial-keyword][pattern]" "true"})))
+      (is (d/refs-match? [coll-iso-sample]
+                         (search/find-refs :collection
+                                           {:spatial-keyword "*aMEricA*"
```

Comment:
I'd drop the wildcard from this search just to make sure that case insensitive aMEricA is matched without setting `options[spatial-keyword][pattern]`.